### PR TITLE
docs(examples): fix grammatical errors

### PR DESCRIPTION
**Pull request by wizardofhogwarts**
**What kind of change does this PR introduce?**

**Did you add tests for your changes?**

**Does this PR introduce a breaking change?**

**What needs to be documented once your changes are merged?**

### CODE REVIEW

File: examples/scope-hoisting/template.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
-This example demonstrates Scope Hoisting in combination with Code Splitting.
+This example demonstrates the Scope Hoisting in combination with Code Splitting.
```
Comment (sokra) [inline examples/scope-hoisting/template.md:1]: That sounds a bit weird.

Comment (wizardofhogwarts) [reply examples/scope-hoisting/template.md]: @sokra i have fixed this.

**Comment by vankop** [comment]
@wizardofhogwarts pls sign CLA

**Comment by alexander-akait** [comment]
@snitin315 Can you check it and resend PR, thank you

**Pull request closed without merging**